packages/cli/src/__tests__/commands/__snapshots__/Format.test.ts.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`format should add a trailing EOL 1`] = `
+exports[`format should have consistent EOL for all lines`] = `
```

Comment:
```suggestion
exports[`format should have consistent EOL for all lines 1`] = `
```
The one is automatically added for the first snapshot here I think and is needed.